test/common/secret/secret_manager_impl_test.cc

```diff
@@ -509,9 +509,9 @@ name: "abc.com.stek"
     name: "abc.com.stek"
     session_ticket_keys:
       keys:
-        - filename: "{{ test_rundir }}/test/extensions/transport_sockets/tls/test_data/ticket_key_a"
```

Comment:
IMHO, this doesn't have to be redacted, since leaking the path doesn't leak too much information, and it helps with debugging, but I imagine that it makes annotations easier to implement, so this is fine.